vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -178,6 +203,145 @@ void flagSpatialCellsForAmrCommunication(const dccrg::Dccrg<SpatialCell,dccrg::C
    return;
 }
 
+void findNeighborhoodCells(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
+                           const CellID startingCell,
```

Comment:
Call this startingCellId to avoid confusion? Or startingCellID or -CID?

Reply:
Fair suggestion.